src/bundle-size.ts

```diff
@@ -37,6 +48,33 @@ function loadBuildManifest(workingDir: string): BuildManifest {
   return JSON.parse(file);
 }
 
+function loadReactLoadableManifest(appChunks: string[], workingDir: string): BuildManifest {
+  const file = fs.readFileSync(
+    path.join(process.cwd(), workingDir, '.next', 'react-loadable-manifest.json'),
+    'utf-8',
+  );
+  const content = JSON.parse(file) as ReactLoadableManifest;
+  const pages = {} as BuildManifest['pages'];
+  Object.keys(content).map((item) => {
+    const fileList = getFiles(content[item]);
+    const uniqueFileList = Array.from(new Set(fileList));
+    pages[item] = uniqueFileList.filter(
+      (file) =>
+        !appChunks.find((chunkFile) => file === chunkFile && !file.includes('/node_modules/')),
+    );
+  });
```

Comment:
The node_modules path is actually in the keys of `react-loadable-manifest.json` so you can filter it out there. Also you can use `.forEach` if you don't care about the return value to be clearer about the intent.
```suggestion
  Object.keys(content).forEach((item) => {
    if (item.includes('/node_modules/')) {
      return;
    }
    const fileList = getFiles(content[item]);
    const uniqueFileList = Array.from(new Set(fileList));
    pages[item] = uniqueFileList.filter(
      (file) => !appChunks.find((chunkFile) => file === chunkFile),
    );
  });
```